Makefile

```diff
@@ -11,6 +11,7 @@ RELEASE_SHA ?= $(shell git rev-parse origin/main)
 OPERATOR_IMG = $(IMAGE_BASE):$(VERSION)
 OPERATOR_BUNDLE=observability-operator.v$(VERSION)
 CONTAINER_RUNTIME := $(shell command -v podman 2> /dev/null || echo docker)
+OSD_E2E_IMG=$(IMAGE_BASE)-osde2e:$(VERSION)
```

Comment:
I'd propose we inlcude `test-harness` in the image name. I [proposed `observability-operator-test-harness`](https://issues.redhat.com/browse/MON-3125?focusedId=22000201&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-22000201), but for explicitness sake maybe even `observability-operator-osde2e-test-harness` makes sense?

Reply:
I felt it verbose but wanted to make `osd` explicit.  If we are to follow their nomenclature, I think we should call it `osde2e-observability-operator-test-harness` see:
```
          - name: ADDON_TEST_HARNESSES
            value: quay.io/rmundhe_oc/osde2e-example-test-harness // update to your test harness image
```

I will follow the same.